### PR TITLE
Enable wasm simd testing under recent versions of node. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: tools/shared.py [hunk 1]
```diff
@@ -94,6 +94,8 @@ def shlex_quote(arg):
 # Switch to shlex.join once we can depend on python 3.8:
 # https://docs.python.org/3/library/shlex.html#shlex.join
 def shlex_join(cmd):
+  if type(cmd) is str:
+    return cmd
```
Comment (tlively) [inline tools/shared.py:98]: Is this related?

**Review by tlively — APPROVED** [review]

File: tools/shared.py (hunk 1 above)
Comment (sbc100) [reply tools/shared.py:98]: Only in the sense that SIMD tests are the only ones that pass strings to `run_process`.  e.g. they do `shared.run_process('thing')`.   I hadn't realized that was valid... but when we print the string version of the process (e.g. in --verbose mode) it was showing up as `t h i n g` :)

**Comment by kripken** [comment]
This appears to be failing on Chromium CI:

https://logs.chromium.org/logs/emscripten-releases/buildbucket/cr-buildbucket/8788252883001154657/+/u/Emscripten_testsuite__core0_/stdout

**Comment by sbc100** [comment]
https://chromium-review.googlesource.com/c/emscripten-releases/+/4289677
